docs/labs/lab02-inventory-operations/get_running_configs_by_time.py

```diff
@@ -23,8 +23,11 @@
         }
     }
     try:
-        resultRunningConfig = clnt.post(uri, data=data)[0]['config']
+        resultRunningConfig = clnt.post(uri, data=data)
+        for idx in resultRunningConfig:
+            if 'config' in idx:
+                result = idx['config']
```

Comment:
In this scenario would it make sense to break the loop once the 'config' key is found and the result is set?